app/utils/mail.server.ts

```diff
@@ -49,7 +49,8 @@ export const sendEmail = async (
 export const sendTeamInviteEmail = async (email: string, app: string) => {
   return await sendEmail(email, "Your invite to POKT Portal", EmailTemplates.TeamInvite, {
     app: app,
-    invite_link: "https://www.portal.pokt.network/dashboard/apps",
+    // TODO: Change to correct invite link
+    invite_link: "https://www.portal.pokt.network/account/apps",
```

Comment:
`https://www.portal.pokt.network/account` ?

Reply:
Do we want to have notifications and invitations on the overview page?